app/models/book.rb

```diff
@@ -0,0 +1,8 @@
+class Book < ApplicationRecord
+  validates :title, presence: true
+  validates :author, presence: true
+  validates :image, presence: true
+  validates :editor, presence: true
+  validates :genere, presence: true
+  validates :year, presence: true
```

Comment:
Normalmente ponermos las validaciones iguales en una sola línea

```suggestion
  validates :title, :author, :image, :editor, :genere, :year, presence: true
```